File: product/src/main/java/com/klcw/app/product/ProductApplication.java
package com.klcw.app.product;

import android.app.Application;

import com.klcw.app.lib.network.NetworkConfig;

/**
 * @author kk
 * @datetime: 2018/10/24
 * @desc:
 */
public class ProductApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        NetworkConfig.setup(this, "prd");
    }
}

File: product/src/main/java/com/kk/app/product/ProductApplication.kt
package com.kk.app.product

import android.app.Application
import com.kk.app.lib.network.NetworkConfig

/**
 * @author kk
 * @datetime: 2018/10/24
 * @desc:
 */
class ProductApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        NetworkConfig.setup(this, "prd")
    }
}